src/Data/Bifunctor/Reverse.hs

```diff
@@ -0,0 +1,43 @@
+{-# LANGUAGE CPP #-}
```

Comment:
CPP isn't used in this module, so this language extension should be unnecessary.